guides/deployment/releases.md

```diff
@@ -192,12 +192,12 @@ COPY priv priv
 # your Elixir templates, you will need to move the asset compilation step
 # down so that `lib` is available.
 COPY assets assets
+COPY lib lib
```

Comment:
The instructions on line 190 already warns the dev when this change is needed. I wouldn't make it default, otherwise we would need to change the instructions to say the opposite. Wdyt?

Reply:
Sounds good. I did not see that comment and I lost some time trying to figure out why tailwind was not loaded. So the DX is not great. Perhaps, we could provide an alternative (ready to use) Dockerfile example. I think there are still people struggling about this. Like a section Dockerfile for PurgeCSS. I think this is good to provide some examples with some tools as Next.js does.